styles/core.css

```
@@ -9,3 +9,23 @@ simplete-suggestions {
 simplete-suggestions:empty {
 	display: none;
 }
+
+simplete-status {
+	clip: rect(0 0 0 0);
+	clip-path: inset(50%);
+	height: 1px;
+	overflow: hidden;
+	position: absolute;
+	white-space: nowrap;
+	width: 1px;
```

for consistency AKA OCD

```suggestion
	position: absolute;
	width: 1px;
	height: 1px;
	overflow: hidden;
	clip: rect(0 0 0 0);
	clip-path: inset(50%);
	white-space: nowrap;
```

PS: I can only guess what this does; visually hiding the element without compromising accessibility? I've lost track of the state of the art on that front.

---

I've updated the order of the CSS properties and added a comment and a link as a reference to the code snippet that I use for hiding content visually: https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html#hiding-content-visually